### PR TITLE
[TTH-117]added a get matches command

### PR DESCRIPTION
**Pull request by ReneeAl**
* **What kind of change does this PR introduce?** (Bug fix, new feature...)
Adds a get matches route that returns the user's pet matches.

* **What is the current behavior? What does the code do before you made the new changes?** 
You can currently only get the animal description from the server.

* **What is the new behavior?**
Calling the /getMatches?id=# command will retrieve the description of the #nt matched animal or an error if they don't exist.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
Have to run "npm install" to add node_modules folder to backend.

* **Other information**:

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -18,4 +18,18 @@ app.get('/getAnimalDescription', function (req, res) {
     })
 })
 
+app.get('/getMatches', function(req, res){
+    axios.get('https://codubee-projects-api.herokuapp.com/animal/getMatches')
```
Comment (roseyume) [inline backend/index.js]: /getMatches doesn't getMatches based on user id given by req in fucntion(req,res). ie. https://codubee-projects-api.herokuapp.com/animal/getMatches?id=1 so when I test it localhost:8080/getMatches always returns no matches.

**Review by roseyume — CHANGES_REQUESTED** [review]
/getMatches needs to make a GET request using the id given by req.body.id

File: backend/index.js (hunk 1 above)
Comment (Codubee) [reply backend/index.js]: I agree with Rosey. I think you have to add a query parameter to your route here. You can take a look at [this](https://github.com/Codubee/Node-Js-API/blob/39babb7639c29b9c0f5fe50ae72f1e7b6e737697/index.js#L37) example as a guide

**Review by Codubee — APPROVED** [review]
LGTM!

**Pull request merged**